stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
+        comodel_name='stock.picking',
+        relation='account_invoice_stock_picking_rel',
```

Comment:
@moylop260 thanks for your review.
Yes I know, the official documentation says:

> The attributes relation, column1 and column2 are optional. If not given, names are automatically generated from model names, provided model_name and comodel_name are different!

but It is not entirely clear to me how table and columns names are generated so that I can use them in post_init_hook.

Any insight would be very useful and highly appreciated.

Thanks.